js/w3c/templates/sotd.html

```diff
@@ -131,8 +131,9 @@
             a group
             {{/if}} operating under the
             <a{{#if doRDFa}} id="sotd_patent" property='w3p:patentRules'{{/if}}
-            href='https://www.w3.org/Consortium/Patent-Policy-20040205/'>5 February 2004 W3C Patent
-            Policy</a>.
+            href='https://www.w3.org/Consortium/Patent-Policy-20170801/'>
+            5 February 2004 (updated 1 August 2017) 
```

Comment:
"W3C Patent Policy" seems to have been swallowed in the update! Text should "5 February 2004 (updated 1 August 2017) W3C Patent Policy"

Reply:
Well spotted, thanks!